### DISCUSSION TITLE
[预览]v2.0.5

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#2592] [分享 / Share]
- 修复 `播放时自动关灯` 对自动播放模式的检测. (PR #2581 by [FoundTheWOUT](https://github.com/FoundTheWOUT))
- 组件注入 commit hash 信息.
- 修复 `简化评论区` 粉丝牌是舰长时没删干净. (#2329)
- 更新了各种功能的描述.
- 修复 `控制栏触摸优化` 在番剧区失效.
- 修复 `直播勋章快速更换` 弹窗溢出, 支持设定最大加载数量. (#2585)
- `下载视频` 支持 8K 清晰度. (#2589)

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.0.5-preview'>[预览]v2.0.5</a>.</em>

**Comment by onionycs** [comment]
source code.zip 包里面没有 2.0.5版本。。。

**Reply by the1812** [reply]
不要下 zip 包, 去 [README](https://github.com/the1812/Bilibili-Evolved/tree/preview) 那下

**Comment by onionycs** [comment]
知道啦～已经安装好了，感觉功能好复杂，有教程吗？












杨崇盛
***@***.***

**Reply by the1812** [reply]
#2448

**Comment by HAOZ666** [comment]
你好，视频播放页夜间模式是不是出bug了？

**Reply by anzuFan** [reply]
我也遇到 视频播放页夜间模式 视频框附近变白的问题了。

**Reply by the1812** [reply]
见 #2597

**Comment by onionycs** [comment]
新版本真好用，谢谢1812的开发~🤭😁

**Comment by onionycs** [comment]
要是能实现这个脚本的功能就好了──『[Bilibili 视频默认选择最高清晰度](https://greasyfork.org/zh-CN/scripts/374770-bilibili-%E8%A7%86%E9%A2%91%E9%BB%98%E8%AE%A4%E9%80%89%E6%8B%A9%E6%9C%80%E9%AB%98%E6%B8%85%E6%99%B0%E5%BA%A6)』 这是一个已经被弃用的项目。。

**Reply by Geeyun-JY3** [reply]
B 站现在是能记忆画质的。你找个 8K 视频然后试试选择最高清晰度？
#1974

**Reply by onionycs** [reply]
谢谢你的回复，我去试试！

**Comment by onionycs** [comment]
![新功能建议](https://user-images.githubusercontent.com/37523020/145149789-8e8ef2a8-2753-4108-9e8b-7ae700987cab.png)
如图示提供了一个建议，因为每一次想知道还剩多少时间都要麻烦的移动鼠标。

**Reply by Geeyun-JY3** [reply]
我的建议是独立开个功能建议的 discussion，窝在这么个版本发布的 discussion 里这个功能的开发者不一定看得见

**Reply by the1812** [reply]
我看得到, 只不过放这里很可能回头就忘了(

**Reply by Geeyun-JY3** [reply]
就是这么个理。而且这个组件的作者还真不一定看得见。
